implementations/kmp-json-schema-validator/gradle.properties

```diff
@@ -0,0 +1,2 @@
+org.gradle.parallel=true
```

Comment:
If it would mean that this file can be removed, I would just use `--parallel` when calling gradle. But it's just my aesthetic preference :)

Reply:
Yes, it will be equivalent. I will remove gradle.properties file)